app/services/copy_vacancy.rb

```diff
@@ -61,4 +63,20 @@ def setup_new_vacancy
     @new_vacancy.total_get_more_info_clicks = 0
     @new_vacancy.total_get_more_info_clicks_updated_at = Time.zone.now
   end
+
+  def setup_organisation_vacancies
+    @vacancy.organisation_vacancies.each do |organisation_vacancy|
+      @new_vacancy.organisation_vacancies.build(organisation: organisation_vacancy.organisation)
+    end
+  end
+
+  def setup_job_location
+    if @new_vacancy.organisation.is_a?(School)
+      @new_vacancy.job_location = 'at_one_school'
+      @new_vacancy.readable_job_location = @new_vacancy.organisation_name
+    else
```

Comment:
If I copy a job that is based at a single school as a SchoolGroup, will this achieve that?

Reply:
we do not have the concept of a vacancy belonging to a school in a school_group. this will simplify things. what was the reason for that?